views/messages.ejs

```diff
@@ -0,0 +1,146 @@
+            </div>
+        </aside>
+        <div class="column is-10">
+            <div class="card-header">
+                <div class="columns is-mobile">
+                    <div class="column is-4 control is-grouped has-text-left">
+                        <a class="ribbon button">
+                            <i class="fa fa-arrow-left"></i>
+                        </a>
+                        <a class="ribbon button">
+                            <i class="fa fa-repeat"></i>
+                        </a>
+                    </div>
+                    <div class="column is-8 control is-grouped is-narrow has-text-right">
+                        <a class="ribbon button">
+                            <i class="fa fa-angle-left"></i>
+                        </a>
+                        <a class="ribbon button">
+                            <h1>1 / 10</h1>
+                        </a>
+                        <a class="ribbon button">
+                            <i class="fa fa-angle-right"></i>
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="headerchat">
+                <div class="card-content">
+                    <div class="columns">
+                        <div class="column is-1">
+                            <input class="star" type="checkbox" title="bookmark page" checked>
+                        </div>
+                        <div class="column is-1">
+                            <span class="msg-from">
+                                <img src="../media/flag/USA.png" alt="USA" width="44px" height="44px" style="border-radius:75%">
+                            </span>
+                        </div>
+                        <div class="column is-9">
+                            <div class="mesg-header">
+                                <span class="mesg-header">
+                                    <h1>USA</h1>
+                                </span>
+                            </div>
+                            <div class="msg-subject">
+                                <span class="msg-subject">
+                                    <small id="subject-0" style="color:antiquewhite">Hello Delegate lets form an alliance and Draft a Resolution</small>
+                                </span>
+                            </div>
+                        </div>
+                        <div class="column is-1">
+                            <span class="msg-timestamp" style="color:antiquewhite">12:34</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <a class="button gold-button is-bold">
+                <span class="compose">
+                    <strong>
+                        REPLY
+                    </strong>
+                </span>
+            </a>
+        </div>
+    </div>
+    <script async type="text/javascript" src="../public/js/bulma.js"></script>

```

Comment:
Let's add the javascript includes in `<head>` tag itself. With proper comments.

Reply:
Sorry, that was included by mistake, its fixed up now (I removed it)